packages/katello/katello/katello.spec

```diff
@@ -172,19 +172,14 @@ Requires: rh-mongodb34
 Requires: foreman-installer-%{name}
 Requires: rubygem-highline
```

Comment:
Why is this still needed?

Reply:
Probably because of https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/katello/katello/hostname-change.rb#L9 but that should live in -common then too.